example/handler.ts

```diff
@@ -1,21 +1,34 @@
 function addNumbers(numbers: Array<number>) {
+  console.log("Adding Numbers")
+  console.log("Numbers to add: " + numbers)
   return numbers.reduce((acc,cv) => acc + cv)
 }
 
 module.exports.add = async (event: any) => {
- //Works locally only. Has a problem behind the API Gateway
-  let bodyObj = event.body
+
+  if (event.body) {
```

Comment:
Not sure if we should use this or `"body" in event`. Not 100% sure about the difference.